bundles/http/Citrix/Slack/2e8a194d-0c01-44c1-9a62-a5ee1d5ecbb3.js

```diff
@@ -0,0 +1,276 @@
+                        }
+
+                        indexUser++
+                    } while (searchText.indexOf('>') >= 0)
+
+                    parsedMessages[indexMessages].datetime = new Date(parsedMessages[indexMessages].ts * 1000);
+                    parsedChannelMessages.push(parsedMessages[indexMessages])
+
+                    indexMessages++;
+                } while (parsedMessages[indexMessages] !== undefined)
+
+
+            } else {
+                throw new Error(`Could not retrieve favorite channel messages (${response.status}: ${response.statusText})`);
+            }
+
+            indexChannels++;
+        } while (parsedFavoriteChannels[indexChannels] !== undefined)
+
+        urlEndpoint = ENDPOINT_CHANNEL;
+
+        let respChannels = await client.fetch(urlEndpoint)
+
+        if (respChannels.ok) {
+            let allChannels = await respChannels.json()
+            let prettyChannels = JSON.stringify(allChannels.channels, null, '\t')
+            let parsedChannels = JSON.parse(prettyChannels)
+            let indexChannels = 0;
+
+            do {
+                let channel = parsedChannels[indexChannels]
+                urlEndpoint = ENDPOINT_CHANNEL_MEMBERS + channel.id
+                let respChannelsMembers = await client.fetch(urlEndpoint)
+
+                if (respChannelsMembers.ok) {
+                    let allChannelsMember = await respChannelsMembers.json()
+                    if (allChannelsMember.members === undefined) {
+                        indexChannels++
+                        continue
+                    }
+                    let prettyChannelsMembers = JSON.stringify(allChannelsMember.members, null, '\t')
+                    let parsedChannelsMembers = JSON.parse(prettyChannelsMembers)
+                    let insert = true;
+                    let indexChannelsMembers = 0;
+                    let channelMember = "";
+
+                    do {
+
+                        channelMember = parsedChannelsMembers[indexChannelsMembers]
+                        insert = true;
+                        channel.is_favorite = false
+                        channel.user = channelMember
+                        channel.ts = null
+                        parsedAllFavoriteChannels.forEach(async (favoriteChannel) => {
+                            if (channel.id === favoriteChannel.text && channelMember === favoriteChannel.user) {
+                                channel.is_favorite = true
+                                channel.ts = favoriteChannel.ts
+                            }
+                        });
+                        let id = channel.id
+                        let user = channel.user
+                        let is_favorite = channel.is_favorite
+                        let name = channel.name
+                        let ts = channel.ts
+                        parsedAllChannels.push({id, user, name, ts, is_favorite})
```

Comment:
IMHO much more readable

```suggestion
                        parsedAllChannels.push({
                              id: channel.id, 
                              user: channel.user,
                              name: channel.name,
                              ts: channel.ts, 
                              is_favorite
                         })
```

Reply:
Done. Way better!